modules/aac/decode.c

```diff
@@ -12,6 +13,13 @@
 #include "aac.h"
 
 
+struct au_hdr {
```

Comment:
perhaps add a comment saying that "AU" means "Access Unit" ?
(au can also mean other things such as "audio")